lib/src/main/scala/spinal/lib/bus/regif/BusIfAdapter/AhbLite3BusInterface.scala

```diff
@@ -22,6 +22,8 @@ case class AhbLite3BusInterface(bus: AhbLite3, sizeMap: SizeMapping, regPre: Str
   val doRead      = bus.HREADY & askRead
 
   val addressDelay = RegNextWhen(bus.HADDR, askRead | askWrite)
+  readError.clearWhen(askWrite)
```

Comment:
The error signal is for the data phase of AHB right ? or is it for the address phase ?

Reply:
Error signal is HRESP, and is for data phase. It is currently set to `False` (never errors).

Btw I think this implementation does not match AHB protocol. Should I PR it?

Reply:
Yes, the AHBLite3BusInterface HRESP not completed here .
In a word, my requirement is that if an error (HRESP/PSLVERROR) occurs in a read/write operation, it should not continue to the next access. The APB3bus interface has such a bug before. At present, it seems that the APB3 problem has been fixed. 

> Btw I think this implementation (outside the scope of this PR) does not match AHB protocol. Should I PR it?

But I haven't finished AHB yet. If you are willing to help me finish it, I will be very grateful